### DISCUSSION TITLE
Cannot run ImageGlass 8.3.11.21 (x86) on Windows 8.1 (x64)

### DISCUSSION THREAD

**Discussion by Anton-V-K** [d2phap/ImageGlass#1213] [General] [answered]
I'm unable to start `ImageGlass.exe` or `igcmd.exe` (from portable ImageGlass 8.3.11.21 (x86)) on Windows 8.1 (x64) - following error is shown:
```
ImageGlass.exe - This application could not be started.
---------------------------
This application could not be started.

Do you want to view information about this issue?
```
Pressing _Yes_ opens the page [Troubleshooting a 'This application could not be started' error message](https://docs.microsoft.com/en-gb/dotnet/framework/install/application-not-started?version=(null)&processName=ImageGlass.exe&platform=0009&osver=6&isServer=0&shimver=4.0.30319.34014).
It seems some specific version of .NET Framework is missing.
Which one is needed for ImageGlass 8.3?

**Comment by fire-eggs** [comment, marked as answer]
Hmmm, it's unfortunate this information is not specified in the download page.

[You need to install .NET 4.8 or later](https://dotnet.microsoft.com/download/dotnet-framework/net48).

**Reply by Anton-V-K** [reply]
Thanks for the reply.
You are right, this information is specified at the bottom of the [download page](https://imageglass.org/release/imageglass-kobe-8-3-11-21-33):

> Requirements
> - Support OS: 7 SP1, 8, 8.1, 10, 11
> - Runtimes: .NET Framework 4.8+

**Comment by fire-eggs** [comment]
thanks for pointing it out - I missed it!!
